src/common/helpers/utils.ts

```diff
@@ -84,4 +84,21 @@ export function generatePhysicalName(
   }
 
   return prefix.toLowerCase() + (lower ? allParts.toLowerCase() : allParts) + '-' + uniqueStackIdPart;
+}
+
+export const maximumLambdaMemorySizeContextItem = 'maximumLambdaMemorySize';
```

Comment:
Do we need this initialization of `maximumLambdaMemorySizeContextItem = 'maximumLambdaMemorySize'; ?specially when  we pull this value from the context .

Reply:
I prefer constants instead of strings. It helps with reuse.